armi/bookkeeping/report/reportingUtils.py

```diff
@@ -139,13 +139,21 @@ def _listInputFiles(cs):
 
         # bonus: grab the files stored in the crossSectionControl section
         for fluxSection, fluxData in cs["crossSectionControl"].items():
+            fNames = []
+            # Users shouldn't ever have both of these defined, but this is not the place
+            # for code to fail if they do. Allow for both to not be None.
             if fluxData.xsFileLocation is not None:
+                # list of files
+                fNames.extend(fluxData.xsFileLocation)
+            if fluxData.fluxFileLocation is not None:
+                # single file
+                fNames.append(fluxData.fluxFileLocation)
```

Comment:
I need confirmation that this is accurate in all cases. Don't want to mess things up if single xsFiles are allowed (single item list will work here)

Reply:
I think it will always be a `list`. Even if a user just wants to provide one file, it should still be a single item `list`.

Reply:
good that's what I thought! thanks for confirming